js/impress.js

```diff
@@ -397,6 +397,33 @@
                 scale:     1
             };
             
+            // Hide the mouse cursor in fullscreenmode after the configured
+            // amount of milliseconds without movement
+            if(rootData.hideMouseAfter) {
+                var mouseMoveTimer = null,
+                isCursorVisible = true;
+                
+                var hideCursor = function() {
+                    //isFullscreen
+                    if(!window.screenTop && !window.screenY){
```

Comment:
This if() is actually a bit obscure. It would be more readable if you implement a helper function isFullscreen(), which returns true when this if is true. (I can get it since there is the comment, but would prefer the helper function, especially as this code moves to a separate plugin, so an extra line of code or two isn't an issue.)